### ISSUE TITLE
Application references version '13.18.0.0' of 'NSwag.Core' which is lower than the required version '13.19.0.0'.

### ISSUE THREAD

**Issue by JanHergenhan** [RicoSuter/NSwag#4419]
Hi,

I recently installed the new nswag.consolecore package (v13.19.0).
Running `nswag run nswag.json /runtime:Net60` results in an error:

```
Application references version '13.18.0.0' of 'NSwag.Core' which is lower than the required version '13.19.0.0'.
System.InvalidOperationException: Swagger generation failed with non-zero exit code '4'.
   at NSwag.Commands.Generation.AspNetCore.AspNetCoreToSwaggerCommand.RunAsync(CommandLineProcessor processor, IConsoleHost host) in /_/src/NSwag.Commands/Commands/Generation/AspNetCore/AspNetCoreToOpenApiCommand.cs:line 231
   at NSwag.Commands.NSwagDocumentBase.GenerateSwaggerDocumentAsync() in /_/src/NSwag.Commands/NSwagDocumentBase.cs:line 275
   at NSwag.Commands.NSwagDocument.ExecuteAsync() in /_/src/NSwag.Commands/NSwagDocument.cs:line 81
   at NSwag.Commands.Document.ExecuteDocumentCommand.ExecuteDocumentAsync(IConsoleHost host, String filePath) in /_/src/NSwag.Commands/Commands/Document/ExecuteDocumentCommand.cs:line 85
   at NSwag.Commands.Document.ExecuteDocumentCommand.RunAsync(CommandLineProcessor processor, IConsoleHost host) in /_/src/NSwag.Commands/Commands/Document/ExecuteDocumentCommand.cs:line 32
   at NConsole.CommandLineProcessor.ProcessSingleAsync(String[] args, Object input)
   at NConsole.CommandLineProcessor.ProcessAsync(String[] args, Object input)
   at NSwag.Commands.NSwagCommandProcessor.ProcessAsync(String[] args) in /_/src/NSwag.Commands/NSwagCommandProcessor.cs:line 61
```

**Comment by vpetrusevici** [comment]
the same. looks like msbuild has strict some dependency here. I still use 13.18.2 to avoid it

**Comment by JanHergenhan** [comment]
Seems to be fixed with version 13.20.0
